Source/ACE.Server/WorldObjects/Hotspot.cs

```diff
@@ -152,32 +152,35 @@ private void Activate(Player player)
             var amount = DamageNext;
             var iAmount = (int)Math.Round(amount);
 
+            var pAmount = 0;
```

Comment:
are 3 amounts really needed here?

the whole point of iAmount was to track the integer amount through the pipeline, for the final output message

Reply:
iAmount wasn't changing on 3 of the 4, i can make it do that of course, but at the same time i didn't see the harm either way